### PR TITLE
Update market creator

### PR DESCRIPTION
**Pull request by jmoreira-valory**
Update market creator to fix issues related to cold starts.

### CODE REVIEW

**Review by 0xArdi — APPROVED** [review]

File: packages/valory/skills/market_creation_manager_abci/models.py [hunk 1]
```diff
@@ -103,11 +103,6 @@ def __init__(self, *args: Any, **kwargs: Any) -> None:
         self.approve_market_event_days_offset = self._ensure(
             "approve_market_event_days_offset", kwargs, type_=int
         )
-        self.approve_market_creator = self._ensure(
-            key="approve_market_creator",
-            kwargs=kwargs,
-            type_=str,
-        )
```
Comment (Adamantios) [inline packages/valory/skills/market_creation_manager_abci/models.py:110]: Not used?

File: packages/valory/skills/market_creation_manager_abci/models.py [hunk 2]
```diff
@@ -152,9 +147,6 @@ def __init__(self, *args: Any, **kwargs: Any) -> None:
         self.market_proposal_round_timeout_seconds_per_day = self._ensure(
             "market_proposal_round_timeout_seconds_per_day", kwargs, type_=int
         )
-        self.market_closing_newsapi_api_key = self._ensure(
-            "market_closing_newsapi_api_key", kwargs, type_=str
-        )
```
Comment (Adamantios) [inline packages/valory/skills/market_creation_manager_abci/models.py:157]: Not used?

File: packages/valory/skills/market_creation_manager_abci/behaviours.py [hunk 3]
```diff
@@ -659,7 +659,8 @@ def _collect_latest_open_markets(
         self, openingTimestamp_gte: int, openingTimestamp_lte: int
     ) -> Generator[None, None, Dict[str, Any]]:
         """Collect FPMM from subgraph."""
-        creator = self.params.approve_market_creator
+        creator = self.synchronized_data.safe_contract_address.lower()
+        self.context.logger.info(f"_collect_latest_open_markets {creator=}")
```
Comment (Adamantios) [inline packages/valory/skills/market_creation_manager_abci/behaviours.py:663]:
```suggestion
        self.context.logger.debug(f"_collect_latest_open_markets {creator=}")
```

File: packages/valory/skills/market_creation_manager_abci/behaviours.py [hunk 4]
```diff
@@ -2257,6 +2259,7 @@ def _get_unanswered_questions(
     ) -> Generator[None, None, List[Dict[str, Any]]]:
         """Collect FPMM from subgraph."""
         creator = self.synchronized_data.safe_contract_address.lower()
+        self.context.logger.info(f"_get_unanswered_questions {creator=}")
```
Comment (Adamantios) [inline packages/valory/skills/market_creation_manager_abci/behaviours.py:2262]:
```suggestion
        self.context.logger.debug(f"_get_unanswered_questions {creator=}")
```

File: packages/valory/skills/market_creation_manager_abci/behaviours.py [hunk 5]
```diff
@@ -1132,10 +1133,10 @@ def get_payload(self) -> Generator[None, None, str]:
 
     def get_markets(self) -> Generator[None, None, Tuple[List[Dict[str, Any]], int]]:
         """Collect FMPMM from subgraph."""
+        creator = self.synchronized_data.safe_contract_address.lower()
+        self.context.logger.info(f"get_markets {creator=}")
```
Comment (Adamantios) [inline packages/valory/skills/market_creation_manager_abci/behaviours.py:1137]:
```suggestion
        self.context.logger.debug(f"get_markets {creator=}")
```

File: packages/valory/skills/market_creation_manager_abci/models.py (hunk 1 above)
Comment (jmoreira-valory) [reply packages/valory/skills/market_creation_manager_abci/models.py:110]: No, it was a legacy parameter.

File: packages/valory/skills/market_creation_manager_abci/models.py (hunk 2 above)
Comment (jmoreira-valory) [reply packages/valory/skills/market_creation_manager_abci/models.py:157]: No, it was a legacy parameter.

File: packages/valory/skills/market_creation_manager_abci/behaviours.py (hunk 3 above)
Comment (jmoreira-valory) [reply packages/valory/skills/market_creation_manager_abci/behaviours.py:663]: I will delete this line, as it's contained in other parts of the logs.

File: packages/valory/skills/market_creation_manager_abci/behaviours.py (hunk 5 above)
Comment (jmoreira-valory) [reply packages/valory/skills/market_creation_manager_abci/behaviours.py:1137]: I will delete this line, as it's contained in other parts of the logs.

File: packages/valory/skills/market_creation_manager_abci/behaviours.py (hunk 4 above)
Comment (jmoreira-valory) [reply packages/valory/skills/market_creation_manager_abci/behaviours.py:2262]: I will delete this line, as it's contained in other parts of the logs.

**Review by Adamantios — APPROVED** [review]

**Pull request merged**